src/Graph.js

```diff
@@ -975,14 +983,29 @@ class NonLinearDemandSupplyGraphAUC extends NonLinearDemandSupplyGraph {
             this.intersection.Y()
         ], invisiblePointOptions);
 
-        if (areaConf !== 5) {
+        if (isVisible) {
             drawLabel(
                 this.board,
                 [p1, p2, this.intersection],
                 this.options.gAreaAName)
         }
 
         this.board.update();
+
+        const maxval = 100;
+        const points = curve.dataX.reduce(function(acc, e, idx) {
+            if (idx % 5 === 0 && idx < curve.dataY.length) {
```

Comment:
I think this might introduce a subtle bug. Thinking about the points that we use to construct this polygon, we need the last two points to be anchored at the intersection, and the last one to be on the y-axis. Maybe we can do mod 5, except if its the last two points.